presto-main/src/main/java/com/facebook/presto/twitter/logging/QueryScriber.java

```diff
@@ -0,0 +1,168 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.twitter.logging;
+
+import com.facebook.presto.event.query.QueryEventHandler;
+import com.google.common.base.Optional;
+import com.twitter.logging.BareFormatter$;
+import com.twitter.logging.Level;
+import com.twitter.logging.QueueingHandler;
+import com.twitter.logging.ScribeHandler;
+import com.twitter.presto.thriftjava.QueryCompletionEvent;
+import com.twitter.presto.thriftjava.QueryState;
+import io.airlift.log.Logger;
+import org.apache.thrift.TBase;
+import org.apache.thrift.TException;
+import org.apache.thrift.TSerializer;
+
+import java.util.Base64;
+import java.util.logging.LogRecord;
+
+/**
+ * Class that scribes query completion events
+ */
+public class QueryScriber implements QueryEventHandler<com.facebook.presto.event.query.QueryCompletionEvent>
```

Comment:
Just noticed that we have same classname for two classes `QueryCompletionEvent`. I wish there was a way to have alias imports like in python or scala. In any case, I suggest these two options:
1. Change name in com.twitter.presto.thriftjava package for this class to something else, like `QueryCompletionEventThrift`... Not very good idea, but one possibility to keep it clean.
2. Have a standard so that we use full namespace of external package, and all internal `com.facebook.presto` packages are imported. This will keep all the imports consistent.